src/dataflow/src/server/boundary.rs

```diff
@@ -219,15 +232,15 @@ mod boundary_hook {
                         None => break,
                         Some((uuid, id)) => {
                             if !self.suppress.contains_key(&(uuid, id)) {
-                                if let Some((source, frontier)) = self.sources.get(&id) {
+                                if let Some((source, persist, frontier)) = self.sources.get(&id) {
                                     let command = StorageCommand::RenderSources(vec![(
                                         format!("TODO"),
```

Comment:
Does this need to be fixed before merge, or is it not actually used anyway important?

Reply:
It should probably be fixed. At least, we should sort out what should go over the wire as part of the request to build a source, and put all the things in there. E.g. a name, an `as_of`, any linear operator stuff.

Reply:
This was fixed. I made up some new names, as the previous name was just the requesting dataflow's name.